input/docs/fundamentals/sharing-build-state.md

```diff
@@ -0,0 +1,106 @@
+Order: 130
+---
+
+Sharing variables and state across your builds can currently be achieved in a handful of different ways, depending on what state you want to share and across what scope.
+
+## Typed Context
+
+**_This only applies to Cake version `0.28.0` and later_**
+
+> For more background on typed context, check out [the release blog post](https://cakebuild.net/blog/2018/05/cake-v0.28.0-released#typed-context)
+
+You can use a typed context to easily share complex state across tasks *without* using global variables or static members. Using typed context in your script is done in 3 parts: the context itself, the `Setup` method, and individual `Task` methods.
+
+### Creating your typed context
+
+First, you will need a class to act as the typed context. This can be any standard C#, and doesn't need to include any Cake-specific code. For this example, we'll use the following simple class:
+
+```csharp
+public class BuildData
+{
+	public string Configuration { get; }
+	public bool BuildPackages { get; }
+    public List<string> Frameworks {get; set;}
```

Comment:
Minor nitpick... Should line up with other members.

Reply:
Good catch, will fix.